web/src/features/fireWeather/components/maps/WeatherMap.tsx

```diff
@@ -44,24 +40,51 @@ const source = new olSource.XYZ({
 
 const zoom = 6
 
+const rhPointStyleFunction = (feature: any) => {
```

Comment:
This any we don't know the type for, I think an eslint exception comment to suppress the warning would help.